### PR TITLE
[4.10] Populate 4.10 r-c configs

### PR DESCRIPTION
**Pull request by petr-muller**
Doing the motions for 4.10 release-controller configs

Using latest 4.9 FC in lieu of a 4.9 stable (there's no stable until GA)

```
    release:
      channel: candidate
      version: 4.9.0-fc.0
```

### CODE REVIEW

**Comment by petr-muller** [comment]
@bradmwilliams @jupierce I think this is complete setup for 4.10, PTAL

The rehearsals are doomed b/c the necessary releases do not exist yet - it's chicken and egg (how do we test 4.10->4.10 upgrades without at least one accepted 4.10 release?)

File: core-services/release-controller/_repos/ocp-4.8-default.repo [hunk 1]
```diff
@@ -30,8 +30,8 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-server-ose]
 name = rhel-server-ose
-baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.8/rhel-server-ose-rpms/
-          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.8/rhel-server-ose-rpms/
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.9/rhel-server-ose-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.9/rhel-server-ose-rpms/
```
Comment (bradmwilliams) [inline core-services/release-controller/_repos/ocp-4.8-default.repo:34]: Hey @petr-muller,
Why are these 4.8 files pointing to 4.9?

Comment (petr-muller) [reply core-services/release-controller/_repos/ocp-4.8-default.repo:34]: Good catch, these should have been the 4.9 files.

Comment (petr-muller) [reply core-services/release-controller/_repos/ocp-4.8-default.repo]: Fixed

**Review by bradmwilliams — APPROVED** [review]

**Comment by petr-muller** [comment]
I needed to update the mappings to pick up a recent image change, no other changes

**Comment by petr-muller** [comment]
Will bot-tag this to merge

**Pull request merged**